### ISSUE TITLE
Binding to ConverterParameter 

### ISSUE THREAD

**Issue by RomanSoloweow** [AvaloniaUI/Avalonia#6840]
```XAML
  <toolBox:ToolBoxItemView  IsVisible="{Binding $parent[views:EventView].EventType,
                                               Converter={StaticResource PopurTypeVisibleConverter}, 
                                               ConverterParameter={Binding $self.Type}}"/>
```

I expect ConverterParameter to work like CommandParameter.
But instead of the value, I get a Binding instance

**Comment by RomanSoloweow** [comment]
![image](https://user-images.githubusercontent.com/29205560/139829741-5c775f2d-7989-4a75-b2df-6d7483c775ce.png)

**Comment by Symbai** [comment]
> I expect ConverterParameter to work like CommandParameter.

Why? It works exactly the same as on WPF and probably also on WinUI. What you ask for is a feature request, not a bug.

**Comment by maxkatz6** [comment]
Yes, it works like that in both WPF and WinUI. Though it would be nice to have anyway.
Right now, it's possible to use MultiBinding instead.

**Comment by RomanSoloweow** [comment]
@maxkatz6 Am I correct in understanding that it is enough to change the ConverterParameter type to DirectProperty for this?

**Comment by RomanSoloweow** [comment]
@grokys, @kekekeks 

Judging by #6865, it seems like I'm not the only one who needs this function.
What do you think about this?

**Comment by AcidJunkie303** [comment]
Ran into the same issue where the ConverterParameter needs to receive a view model instance.

**Comment by dojo90** [comment]
I would also be happy if this function would be implemented. 👍

**Comment by MarcoMuellner** [comment]
> Yes, it works like that in both WPF and WinUI. Though it would be nice to have anyway. Right now, it's possible to use MultiBinding instead.

Well yes and no. MultiBinding doesn't allow TwoWay, nor gives you a ConvertBack method, so its scope is a bit limited. Personally, i would need exactly that, and it would be perfectly fulfilled through the binding of the ConverterParameter.

**Comment by Energizer188** [comment]
I ran into the same issue, anyone knows how to extract the actual value from the binding instance?
It would be nice indeed to implement this feature

**Comment by maxkatz6** [comment]
> extract the actual value from the binding instance?

You can't. For now, use MultiBinding if you don't need convert back value.

**Comment by Mrxx99** [comment]
That this is not possible always frustrated me in WPF. It is so unintuitive. It would be great if Avalonia could be better there.

**Comment by timunie** [comment]
@Mrxx99 it's planned to have this at a certain point im time. We also have PR open for it which you can test if you want to. But I can't give you a timing when it will be available as other things have higher priority

**Comment by SuperJMN** [comment]
Getting frustrated by this one. Is there a plan for this. It's quite counterintuitive.

**Comment by AlexeyTliss** [comment]
This issue makes ObjectConverters basically unusuable

I propose to either support converter parameter bindings or at least change ObjectConverter to IMultiValueConverter

**Comment by cyraid** [comment]
Yeah I just got hit with this now too. I'm sure not too many people would want to use the parameter with static values. Avalonia could definitely shine here.
